.pre-commit-config.yaml

```diff
@@ -6,7 +6,6 @@ repos:
       - id: trailing-whitespace
       - id: end-of-file-fixer
       - id: check-docstring-first
-      - id: check-yaml
```

Comment:
I am curious.... was this `yaml` pre-commit hook causing problems for it to be removed?

Reply:
Yes, it was causing problems with the example yaml files.  It doesn't like one of the things that yamale expects.